### DISCUSSION TITLE
Known issue on Redpanda image for kafka-client devservice

### DISCUSSION THREAD

**Discussion by ozangunalp** [quarkusio/quarkus#45030] [Community]
Due to the renaming of Redpanda organization from `vectorized` to `redpanda-data`, Github repositories and Docker Hub images have been moved. 

This was undetected before, as both Github and Docker provided redirects for repository URLs. 

Effective today 10 December 2024 ~GMT, Docker Hub redirects has been removed, so Quarkus users of `kafka-client` redpanda devservice would experience the issue https://github.com/quarkusio/quarkus/issues/45026 as the https://hub.docker.com/r/vectorized/redpanda returns 404. 

The [fix](https://github.com/quarkusio/quarkus/pull/45028) has been merged on Quarkus main, setting the default redpanda devservice image URL to `docker.io/redpandadata/redpanda`. 

Workarounds are : 
- Forcing the image-name in application configuration 
  `quarkus.kafka.devservices.image-name=docker.io/redpandadata/redpanda:v24.1.2`
- Re-tagging the image locally 
  ```
  docker|podman pull docker.io/redpandadata/redpanda:v24.1.2
  docker|podman image tag docker.io/redpandadata/redpanda:v24.1.2 docker.io/vectorized/redpanda:v24.1.2
  ```

**Comment by adripace** [comment]
Hello, i recently had the same issue, and even using those workarounds i didn't manage to solve it.
I solved it by  **Forcing the image-name in application configuration** and using the latest version, but maybe other versions works correctly.
Also, for who is reading now check if there are multiple profiles concurrently starting up redpanda.
